velox/exec/Task.h

```diff
@@ -269,6 +269,10 @@ class Task : public std::enable_shared_from_this<Task> {
   /// immediately realized.
   ContinueFuture taskCompletionFuture();
 
+  /// Returns a future which is realized when the task has been deleted, and all
```

Comment:
/// Returns a future which is realized after
/// 1. the task is deleted
/// 2. all the background activities are finished.